### ISSUE TITLE
No poster found for Plex created collections

### ISSUE THREAD

**Issue by darkknight6722** [ShaneIsrael/plex-auto-genres#9]
I already have some collections created by plex and plex auto collections. It shows no posters found for those collections. I don`t want to change the posters for those collections but want posters for new genre collections. Any ways or I am doing something wrong?

**Comment by ShaneIsrael** [comment]
So I've never used plex auto collections and this tool isn't necessarily designed to work with that one. 

If you are seeing the "No posters found" message you don't have anything to worry about. All that is saying is that you don't have a poster in the `/posters` folder that matches that collection.

So for example, say you have a collection in plex named `my awesome collection` which was created by plex auto collections. If you do not want that poster to be updated then you don't have to do anything. However, if you did want the poster for that collection to be update. Place an image in the `/posters` folder under the correct type named `my-awesome-collection.png` the tool will see that image and upload it. 

So basically, if you don't want posters to change for those collections. Make sure you remove any posters from the `/posters` folder that match the titles of the collections you don't want updated. 

Hopefully that makes sense.

**Comment by ShaneIsrael** [comment]
If you have a collection named `adventure` it will match to a poster image named `adventure.png`
If you have a collection named `science fiction` it will match to a poster named `science-fiction.png`

Spaces get replaced with dashes.

**Comment by darkknight6722** [comment]
If i use `--set-posters` it just stops after telling no poster found for already presenting collections. I want it to add posters for new genre collections.

**Comment by ShaneIsrael** [comment]
If you want the new genre collections to be created, you need to first run the script without the `--set-posters` flag. Once its done creating the genre collections for your media you can then run it again with the set posters flag at that point it will see them and upload the matching art.

**Comment by darkknight6722** [comment]
I just found that a minute. Sorry. Everything working fine.
